src/pages/Index.tsx

```
@@ -1,11 +1,94 @@
+        {/* Right: Booking panel */}
+        <div className="md:w-1/3 w-full p-6 border-t md:border-t-0 md:border-l border-gray-100 flex flex-col justify-start">
+          <div className="bg-gray-50 rounded-xl p-5 shadow-sm">
+            <div className="mb-4 text-gray-700 font-semibold">Add dates for price</div>
+            {/* Date picker */}
+            <div className="mb-3">
+              <label className="block text-sm font-medium mb-1">Select dates</label>
+              <Calendar
+                mode="range"
+                selected={dateRange}
+                onSelect={(range) => {
+                  if (range && range.from && range.to) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)